src/Articles.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from "react"
+import getArticles from "./api"
+
+const Articles = ()=>{
+
+    const[articles, setArticles] = useState([])
+
+    useEffect(() => {
+    getArticles().then((articleData) => {
+        setArticles(articleData.articles);
+    })
+    }, []);
+
+    return(
+        <ul className="articlesList">
+            {articles.map((article)=>{
+                return(<><li className="article_list_item">{article.title}</li><img className="article_img" src = {article.article_img_url}/> <p>{article.author}</p></>)})
+            }
```

Comment:
It would be nice to extract the code for how each article is displayed in to an ArticleCard component which you could then render in here.